openapi-generator/src/main/kotlin/org/jellyfin/openapi/Main.kt

```diff
@@ -12,7 +12,7 @@ fun main(vararg args: String) {
 	val modelsOutputDir by arguments
 
 	// Read OpenAPI json
-	val openApiJson = File(openApiFile).readText()
+	val openApiJson = File(openApiFile).readText(Charsets.UTF_8)
```

Comment:
That's the default, so theoretically not needed.

Reply:
Oh that's a leftover from testing let me remove it